src/python/pants/build_graph/app_base.py

```diff
@@ -268,11 +251,7 @@ def globs_relative_to_buildroot(self):
             fileset = bundle.fileset
             if fileset is None:
                 continue
-            elif hasattr(fileset, "filespec"):
-                globs += bundle.fileset.filespec["globs"]
-            else:
-                # NB(nh): filemap is an OrderedDict, so this ordering is stable.
```

Comment:
The comment is still true, but irrelevant now. Dictionaries are always ordered in Python 3.6+.